core/src/main/scala/org/apache/spark/Dependency.scala

```diff
@@ -206,6 +206,21 @@ class ShuffleDependency[K: ClassTag, V: ClassTag, C: ClassTag](
     finalizeTask = Option(task)
   }
 
+  // Set the threshold to 1 billion which represents approximately 1GB of memory
+  // allocated to map output statuses.
+  // A large number of shuffle blocks may crash the driver with an OOM error.
+  private val SHUFFLE_BLOCK_NUMBER_WARNING_THRESHOLD: Long = 1000000000L
```

Comment:
Just curious why this is 1000 based but not 1024 based. I see the comment says `approximately` but why just not make it accurate?